source/guide_hugo.rst

```diff
@@ -0,0 +1,210 @@
+.. highlight:: console
+
+.. author:: Christian Kantelberg <uberlab@mailbox.org>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/hugo-logo.png
+      :align: center
+
+####
+Hugo
+####
+
+Hugo is a fast and modern static site generator written in Go, and designed to make website creation fun again.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * Git_
+  * Domains_
+  * supervisord_
+
+License
+=======
+
+Hugo v0.15 and later are released under the Apache 2.0 license. 
+Earlier versions of Hugo were released under the Simple Public License.
+
+All relevant legal information can be found here 
+
+  * https://opensource.org/licenses/Simple-2.0
+
+Prerequisites
+=============
+
+Your URL needs to be setup:
+
+::
+
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Step 1
+------
+
+Preparing to set up on the Uberspace various directories for further action.
+
+::
+
+ [isabell@stardust ~]$ mkdir -p apps/hugo_render
+ [isabell@stardust ~]$ mkdir hugo_website
+ [isabell@stardust ~]$ 
+ 
+Step 2
+------
+
+To run Hugo from any directory, add the path to Hugo in PATH. To do this, open ``~/.bashrc`` with an editor of your choice (e.g. vim, nano) and add the following line at the end:
+
+::
+
+ export PATH=$HOME/apps/hugo_render/hugo:$PATH
+
+Then re-read your ``.bashrc`` with ``source ~/.bashrc`` to apply the change.
+
+Step 3
+------
+
+Now change the directory ``~/apps/hugo_render`` and set up Hugo.
+
+::
+
+ [isabell@stardust ~]$ cd apps/hugo_render
+ [isabell@stardust hugo_render]$ wget https://github.com/spf13/hugo/releases/download/hugo_42.23.1_Linux-64bit.tar.gz
```

Comment:
If you use a dummy version number, mention this and provide a source for the real URL